pennylane/optimize/rotoselect.py

```diff
@@ -69,7 +69,7 @@ class RotoselectOptimizer:
     ...     generators[0](params[0], wires=0)
     ...     generators[1](params[1], wires=1)
     ...     qml.CNOT(wires=[0, 1])
-    ...     return qml.expval(qml.PauliX(0)), qml.expval(qml.PauliY(1))
+    ...     return qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliX(1))
```

Comment:
(Minor suggestion): on line 77-78 the renamings of `X_1` -> `Z_1` and `Y_2` -> `X_2` could be applied